tests/phpunit/CRM/Lineitemedit/Form/EditTest.php

```diff
@@ -309,8 +308,8 @@ public function testParticipantRecordOnLineItemEdit() {
     $form->setCustomDataTypes();
 
     $form->submit(array(
-      'register_date' => 'now',
-      'register_date_time' => '00:00:00',
+      'register_date' => date('Y-m-d H:i:s'),
+      'receive_date' => date('Y-m-d H:i:s'),
```

Comment:
Test failed because it was trying to insert a 0 for transaction date this fixed it